klock/src/commonMain/kotlin/com/soywiz/klock/DateTimeTz.kt

```diff
@@ -24,6 +24,12 @@ class DateTimeTz private constructor(
         /** Creates a new local [DateTimeTz] from a [unix] time */
         fun fromUnixLocal(unix: Double): DateTimeTz = DateTime(unix).localUnadjusted
```

Comment:
Do you think it makes sense to deprecate this? Could it be confusing to other people?

Reply:
Absolutely!